influxql/ast.go

```diff
@@ -642,6 +642,36 @@ type SelectStatement struct {
 	FillValue interface{}
 }
 
+// HasDerivative returns true if one of the field in the statement is a
+// derivative aggregate
+func (s *SelectStatement) HasDerivative() bool {
+	for _, f := range s.Fields {
+		if strings.HasSuffix(f.Name(), "derivative") {
```

Comment:
That works but using: `"value_derivative": 0.64` currently breaks it.  Will fix that.